NSS-Day-Cohort-42/rare-api-fakes-news
# create a new post & delete a post

## Description
From Postman, you can now: 
- POST a new post object
- DELETE an existing post object

## Tickets
_My PR references these tickets (epic and task tickets)_

1. ​#138 - create a post  (corresponding epic #125)
1. #141 - delete one of my posts (corresponding epic #120)
​
## How do I access this branch?

1. ```git fetch --all```
1. ```git checkout hn-post-create-delete```
1. ```python manage.py runserver``` 
​
## Types of changes

## Testing
- [x] Tested in Postman
- [ ] Tested with Client


``` if necessary, postman code here ```

TO TEST CREATE: 
From Postman:
1. GET http://localhost:8000/posts
1. Select: Body, raw, and JSON (from drop-down)
1. Paste the below object
1. Send
1. GET http://localhost:8000/posts
1. Confirm that the new post object has been added

```{
    "category_id": 1,
    "title": "Fourth Post Title Test",
    "publication_date": "2020-11-11",
    "image_url": "www.google.com",
    "content": "content check 1-2-3"
}```


TO TEST DESTROY: 
From Postman:
1. DELETE http://localhost:8000/posts/[your object id here]
1. GET http://localhost:8000/posts
1. Confirm that the selected post object is gone



​

## Checklist for Reviewer
- [ ] check it out
- [ ] test all edge cases listed and more if you think of them (postman or browser)
- [ ] request changes/approve
- [ ] review code to make sure it fits coding style of project
